ci/scripts/build-with-meson.sh

```diff
@@ -66,7 +66,7 @@ function main() {
     pushd "${SANDBOX_DIR}"
 
     show_header "Run test suite"
-    meson configure -Dtests=true -Db_coverage=true
+    meson configure -Dtests=true -Db_coverage=true -Dipc=true -Ddevice=true
```

Comment:
Missed this on the IPC change. If we want to expand this job in the future would need to accept these options dynamically, but this should work for now